web/public/source/scripts/app.js

```diff
@@ -32,14 +33,21 @@
         'app.services'
     ]).
 
-    run(['$rootScope', 'entries', 'client', function ($rootScope, entries, client) {
-        var defaultUser = {
+    run(['$rootScope', '$cookies', 'entries', 'client', function ($rootScope, $cookies, entries, client) {
+        $rootScope.defaultUser = {
             name: "Guest"
         };
-        $rootScope.defaultUser = defaultUser;
-        $rootScope.selectedUser = defaultUser;
-        entries.getUserEntries(defaultUser);
+        
+        
         client.load();
+        $rootScope.selectedUser = $cookies.getObject('selectedUser');
+        
+        if ( !$rootScope.selectedUser ) {
+            $rootScope.selectedUser = $rootScope.defaultUser;
+            $cookies.putObject("selectedUser", $rootScope.selectedUser);
+        }
+        
+        entries.getUserEntries($rootScope.selectedUser);
```

Comment:
This won't trigger getUserEntries for the default user.
Your should probably do something like this:
```
if ( !$rootScope.selectedUser ) {
    $rootScope.selectedUser = $rootScope.defaultUser;
    $cookies.putObject("selectedUser", $rootScope.selectedUser);
    entries.getUserEntries($rootScope.selectedUser);
} else {
    entries.getUserEntries($rootScope.defaultUser);
}
```